zh/pause-sync-of-tidbcluster.md

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
Suggested filename: `pause-sync-of-tidb-cluster.md`
Because `tidbcluster` doesn't make sense to the search engine, it may affect the page's SEO ranking.

Reply:
changed!